### ISSUE TITLE
Bug: Cmd+A in a comment selects all boxes, not the text in the comment

### ISSUE THREAD

**Issue by 0x1a1a1a** [pure-data/pure-data#2501]
Platform: macOS Sequoia 15.2
Version: 0.55.2

## Steps to reproduce
1. Add a box (e.g. an “Object” box)
2. Add a “Comment” box
3. Enter text in a “Comment” box
4. Press Command+A key combination

---

## Expected result
1. Entered text in a “Comment” box is selected

---

## Actual result
1. A box and a “Comment” box are selected

**Comment by umlaeute** [comment]
well, this is how Pd currently behaves: <kbd>Ctrl</kbd>-<kbd>a</kbd> selects the entire content of the current canvas.

why do you think otherwise? (e.g.: "because the docs say so". or "because this is how other applications, like *Safari* behave". or something else...)

**Comment by porres** [comment]
yes, this is expected, if you want to select the whole text, just click on the comment while in edit mode and you select the whole text

**Comment by porres** [comment]
on the other hand, it is consistent to how GUI (number/symbol/list) boxes behave, where you have to double click to select.

I can see the expectation of that working for comments though, but maybe I'm just used to selecting the whole text by clicking like that and we could just document it... and well, if that changes, I think that the behaviour should not change for GUI boxes... right?
